src/algorithm/evaluation/config.py

```diff
@@ -1014,6 +1017,12 @@ def conditionally_print_explanation(self):
     def get_should_explain_scoring(self):
         return self.should_explain_scoring
 
+    def get_should_export_csv(self):
+        return self.should_export_csv
```

Comment:
Rename to `should_export_explanation_csv` for better readability and avoid confusion with other csvs in the system